src/py-opentimelineio/opentimelineio-bindings/otio_utils.cpp

```diff
@@ -121,34 +120,6 @@ AnyDictionary py_to_any_dictionary(py::object const& o) {
     return safely_cast_any_dictionary_any(a);
 }
 
-std::vector<SerializableObject*> py_to_so_vector(pybind11::object const& o) {
-    if (_value_to_so_vector.is_none()) {
-        py::object core = py::module::import("opentimelineio.core");
-        _value_to_so_vector = core.attr("_value_to_so_vector");
-    }
-
-    std::vector<SerializableObject*> result;
-    if (o.is_none()) {
-        return result;
-    }
-
-    /*
-     * We're depending on _value_to_so_vector(), written in Python, to
```

Comment:
I am not concerned about this comment. I believe it is referring to issues with bopping things back and forth to pyobject, and losing track of reference counts. As there are no unit tests to back up the note about crashing, (what was crashing? no way to tell), I think we have to move ahead assuming that the pybind11 explicit typing strategy is going to fundamentally change these assumptions. And if crashes do occur, we'll need unit tests to demonstrate a fix against.